test/NameBinding/dynamic-callable.swift

```diff
@@ -0,0 +1,396 @@
+// RUN: %target-swift-frontend -typecheck -verify %s
```

Comment:
I would instead put this in test/attr/

Reply:
Done in https://github.com/apple/swift/pull/16980/commits/1329ba1ad003d1706a66dd298d2079ea24f57d81.

I assume `test/NameBinding/dynamic-member-lookup.swift` should also be moved to `test/attr`. I'll remember to do so in a later PR (when I unite code between `@dynamicCallable` and `@dynamicMemberLookup`).